transit_odp/browse/views/timetable_views.py

```diff
@@ -469,17 +527,31 @@ def get_context_data(self, **kwargs):
                 kwargs["booking_arrangements"] = booking_arrangements_info[0][0]
                 kwargs["booking_methods"] = booking_arrangements_info[0][1:]
 
-        # Initial commit for timetable visualiser
-        target_date = datetime.strptime("09/01/2023", "%d/%m/%Y").date()
-        public_use_check_flag = True
-        kwargs["timetable_visualiser"] = TimetableVisualiser(
+        date = self.request.GET.get("date", datetime.now().strftime("%Y-%m-%d"))
+        target_date = datetime.strptime(date, "%Y-%m-%d").date()
+        timetable_inbound_outbound = TimetableVisualiser(
```

Comment:
Return empty data when feature flag is off

Reply:
The flag is based on the per file and if the service is not true then it will automatically remove those services/journeys